### PR TITLE
Added == comparison between YouTube objects.

### PR DESCRIPTION
**Pull request by Gcat101**
That's it.

### CODE REVIEW

File: pytube/__main__.py [hunk 1]
```diff
@@ -91,6 +91,11 @@ def __init__(
     def __repr__(self):
         return f'<pytube.__main__.YouTube object: videoId={self.video_id}>'
 
+    def __eq__(self, o: object) -> bool:
+        # Compare types and urls, if they're same return true, else return false.
+        if type(o) == type(self) and o.watch_url == self.watch_url: return True
+        else: return False
```
Comment (glubsy) [inline pytube/__main__.py]: Why not use the following instead?

`return type(o) == type(self) and o.watch_url == self.watch_url`

Comment (Gcat101) [reply pytube/__main__.py]: oh yeah, sorry i am dumb

**Comment by Gcat101** [comment]
fixed

**Comment by RONNCC** [comment]
hey @Gcat101  is this PR still relevant?

**Comment by Gcat101** [comment]
Yes

**Pull request merged**